content/tracing/advanced_usage/_index.md

```diff
@@ -277,6 +277,20 @@ if (($span = GlobalTracer::get()->getActiveSpan()) !== null){
 
 **Note**: `GlobalTracer::get()->getActiveSpan()` returns `null` if there is no active span.
 
+{{% /tab %}}
+{{% tab "C++" %}}
+
+Add tags directly to a Span object by calling Span.SetTag(). For example:
```

Comment:
```suggestion
Add tags directly to a span object by calling `SetTag()`. For example:
```

I want to lowercase "span". Also, is it `Span.SetTag()` here or just `SetTag()`? (forgive my lack of familiarity with c++)

Reply:
So here I use a capitalised Span because I'm referring to a code object. I suppose in that case I should say `Span` or span (but not Span or `span`). 

I think given how many times I have to say "span", we don't need to make it an inline code block and we should go with "span".

For SetTag, either `SetTag()` or `Span::SetTag` is fine, the latter is probably better.

I've made both these changes in this file, but I'm happy to change it further if you have a different suggestion :)